src/Arcus.EventGrid.Tests.Unit/Parsing/EventGridEventBuilderTests.cs

```diff
@@ -0,0 +1,262 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Arcus.EventGrid.Contracts;
+using Arcus.EventGrid.Parsers;
+using Arcus.EventGrid.Tests.Core.Events;
+using Arcus.EventGrid.Tests.Core.Events.Data;
+using Arcus.EventGrid.Tests.Unit.Artifacts;
+using CloudNative.CloudEvents;
+using Microsoft.Azure.EventGrid.Models;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using Xunit;
+
+namespace Arcus.EventGrid.Tests.Unit.Parsing
+{
+    public class EventGridEventBuilderCloudEventTests
+    {
+        [Fact]
+        public void ParseString_ValidStorageBlobCreatedCloudEvents_ShouldSucceed()
```

Comment:
I did want to reduce the duplication, but that ended up with a functional approach, and I guessed it would be better to explicit state the diff input formatted for each output format.